### PR TITLE
ARS-483: Add ETMP trader details models

### PR DESCRIPTION
**Pull request by koladeadewuyi**
This PR:
- Adds the models for the ETMP trader details
- Scala formatter config

Please note that only the fields below are read from the ETMP `TraderDetailsResponse`

- EORINo
- CDSFullName
- streetAndNumber
- city
- countryCode
- postalCode

### CODE REVIEW

File: .scalafmt.conf [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+style = defaultWithAlign
+maxColumn=120
+align = more
+align.tokens.add = [ ":", ":=", "=" ]
+importSelectors = singleLine
+trailingCommas = never
+verticalMultiline.atDefnSite = true
+verticalMultiline.arityThreshold = 4
+verticalMultiline.newlineAfterOpenParen = true
+verticalMultiline.newlineBeforeImplicitKW = true
```
Comment (C-Thomp) [inline .scalafmt.conf:12]: We have this conf in frontend that we think should work nicely :)

```suggestion
version = 3.6.1
runner.dialect = scala213
lineEndings = unix
importSelectors = singleLine
maxColumn = 100

project {
  git = true
}

align {
  preset = most
  tokens = [ {code = "=>", owner = "Case|Type.Arg.ByName"}, "=", "<-", "->", "%", "%%", "should", "shouldBe", "shouldEqual", "shouldNot", "must" ]
  arrowEnumeratorGenerator = true
  openParenCallSite = false
  openParenDefnSite = false
}

continuationIndent {
  callSite = 2
  defnSite = 2
}


spaces {
  inImportCurlyBraces = false
  beforeContextBoundColon = Never
}

newlines {
  beforeCurlyLambdaParams = multiline
  afterCurlyLambdaParams = squash
}

rewrite {
  rules = [RedundantBraces, Imports],
  imports.sort = scalastyle,
  imports.groups = [
    ["java\\..*", "javax\\..*"],
    ["scala\\..*"],
    ["play\\..*", "uk.gov.hmrc\\..*"],
    ["cats\\..*"],
    [".*"]
  ]
}
```

**Review by RawToast — COMMENTED** [review]
Looking good 🥳

A quick pull / compile reveals a few files have copyright headers. Some of these may just be files generated initially, but the headers will keep on getting added until we replace those endpoints

Happy to approve if those will be generated/committed in the next PR with the Bobby warnings

**Comment by koladeadewuyi** [comment]
> Looking good 🥳
> 
> A quick pull / compile reveals a few files have copyright headers. Some of these may just be files generated initially, but the headers will keep on getting added until we replace those endpoints
> 
> Happy to approve if those will be generated/committed in the next PR with the Bobby warnings

@RawToast, we wanted to keep the changes focused so decided we will address those in a subsequent PR. Thanks

**Review by C-Thomp — APPROVED** [review]
🕺 looks good!

**Pull request merged**